main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -28,59 +31,179 @@ import scala.collection.mutable
 
 object Bootstrap {
   def main(args: Array[String]): Unit = {
+
     // TODO: Do ad-hoc testing here.
     val b = new Bootstrap
-    b.bootstrap("path-to-flix-project")
+    b.bootstrap(".")
+
+    val f = new Flix
+    b.reconfigureFlix(f)
+    println(b.timestamps)
```

Comment:
Speaking of testing, did you try to call f.compile() to see that it works?

Reply:
It works :)